src/main/java/attendance/domain/AttendanceRecord.java

```diff
@@ -0,0 +1,73 @@
+package attendance.domain;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.Month;
+import java.util.Objects;
+
+public final class AttendanceRecord {
+
+    private final String nickname;
+    private final LocalDateTime arrivalDateTime;
+    private final AttendanceType type;
+
+    public AttendanceRecord(String nickname, LocalDateTime arrivalDateTime, AttendanceType type) {
+        this.nickname = nickname;
+        this.arrivalDateTime = arrivalDateTime;
+        this.type = type;
+    }
+
+    public boolean checkSameDate(LocalDate date) {
+        return arrivalDateTime.toLocalDate().equals(date);
+    }
+
+    public boolean isExpulsion() {
+        return type == AttendanceType.EXPULSION;
+    }
+
+    public boolean isInMonth(int year, Month month) {
+        return arrivalDateTime.getYear() == year && arrivalDateTime.getMonth() == month;
+    }
+
+    public boolean isInPeriod(LocalDate startDate, LocalDate endDate) {
+        LocalDate date = arrivalDateTime.toLocalDate();
+        boolean isAfterOrEqual = date.isAfter(startDate) || date.equals(startDate);
+        boolean isBeforeOrEqual = date.isBefore(endDate) || date.equals(endDate);
+        return isAfterOrEqual && isBeforeOrEqual;
+    }

```

Comment:
```suggestion
public boolean isInPeriod(LocalDate startDate, LocalDate endDate) {
        return !arrivalDateTime.toLocalDate().isBefore(startDate) &&
                !arrivalDateTime.toLocalDate().isAfter(endDate);
    }

```
이런 식으로도 줄여볼 수 있을 것 같아요!
지금은 조건이 너무 많은 것 같아서..

Reply:
훨씬 깔끔한 것 같습니다! 바로 리펙토링 해보겠습니다!